docs/developer/contributing/development-unit-tests.asciidoc

```diff
@@ -20,11 +20,13 @@ yarn test:mocha
 == Jest
 Jest tests are stored in the same directory as source code files with the `.test.{js,mjs,ts,tsx}` suffix.
 
-*Running Jest Unit Tests*
+Each plugin and package contains it's own `jest.config.js` file to define it's root and any overrides 
```

Comment:
Typo in two places: it's -> its

Reply:
https://github.com/elastic/kibana/pull/77894/commits/6296df84853f3057f186d46e72912d3fc8295cf4#diff-75718ec4e98fd443b4d8c8618b783558cde16174cd7b3378a64391af96d86caa